app/src/layouts/settingsLayout/settingsLayout.jsx

```diff
@@ -15,26 +15,130 @@
  */
 
 import React from 'react';
+import { defineMessages, useIntl } from 'react-intl';
 import PropTypes from 'prop-types';
 import classNames from 'classnames/bind';
+import { connect } from 'react-redux';
+import { referenceDictionary } from 'common/utils';
+import { uiBuildVersionSelector } from 'controllers/appInfo';
+import { instanceTypeSelector } from 'controllers/appInfo/selectors';
+import { EPAM, SAAS } from 'controllers/appInfo/constants';
 import styles from './settingsLayout.scss';
 
 const cx = classNames.bind(styles);
+const messages = defineMessages({
+  github: {
+    id: 'SettingsLayout.github',
+    defaultMessage: 'Fork us on GitHub',
+  },
+  slack: {
+    id: 'SettingsLayout.slack',
+    defaultMessage: 'Chat with us on Slack',
+  },
+  contactUs: {
+    id: 'SettingsLayout.contactUs',
+    defaultMessage: 'Contact us',
+  },
+  epam: {
+    id: 'SettingsLayout.epam',
+    defaultMessage: 'EPAM',
+  },
+  documentation: {
+    id: 'SettingsLayout.documentation',
+    defaultMessage: 'Documentation',
+  },
+  privacyPolicy: {
+    id: 'SettingsLayout.privacyPolicy',
+    defaultMessage: 'Privacy Policy',
+  },
+  rights: {
+    id: 'SettingsLayout.rights',
+    defaultMessage: '© Report Portal 2023 All rights reserved',
+  },
+  build: {
+    id: 'SettingsLayout.build',
+    defaultMessage: 'Build: {buildVersion}',
+  },
+});
 
-export const SettingsLayout = ({ navigation, children }) => (
-  <div className={cx('container')}>
-    {navigation && <div className={cx('navigation')}>{navigation}</div>}
-    <div className={cx('section')}>{children}</div>
-  </div>
-);
+const SettingsLayoutBase = ({ navigation, children, buildVersion, instanceType }) => {
+  const { formatMessage } = useIntl();
 
-SettingsLayout.propTypes = {
+  const footerLeftItems = [
+    formatMessage(messages.build, {
+      buildVersion,
+    }),
+    formatMessage(messages.rights),
+  ];
+
+  const footerRightItems = [
+    <a href={referenceDictionary.rpGitHub} target="_blank" className={cx('link')}>
+      <span>{formatMessage(messages.github)}</span>
+    </a>,
+    <a href={referenceDictionary.rpSlack} target="_blank" className={cx('link')}>
+      <span>{formatMessage(messages.slack)}</span>
+    </a>,
+    <a href={referenceDictionary.rpEmail} target="_blank" className={cx('link')}>
+      <span>{formatMessage(messages.contactUs)}</span>
+    </a>,
+    <a href={referenceDictionary.rpEpam} target="_blank" className={cx('link')}>
+      <span>{formatMessage(messages.epam)}</span>
+    </a>,
+    <a href={referenceDictionary.rpDoc} target="_blank" className={cx('link')}>
+      <span>{formatMessage(messages.documentation)}</span>
+    </a>,
```

Comment:
You can build that elements during config mapping and left only messages and links in the config.